Tests/CreateAPITests/GenerateOptionsTests.swift

```diff
@@ -537,4 +537,61 @@ final class GenerateOptionsTests: GenerateTestCase {
             """
         )
     }
+    
+    func testPetstoreSPMImports() throws {
+        try snapshot(
+            spec: .petstore,
+            name: "petstore-SPM-imports",
+            testCompilationOnLinux: false, // custom imports aren't available there.
+            arguments: [
+                "--package", "petstore-SPM-imports"
+            ],
+            configuration: """
+            package:
+                dependencies:
+                - url: https://github.com/apple/swift-argument-parser
+                  module: ArgumentParser
+                  rule:
+                      exact:
+                          version: 1.1.1
+                - url: https://github.com/apple/swift-atomics
+                  module: Atomics
+                  rule:
+                      upToNextMajor:
+                          from: 1.0.0
+                - url: https://github.com/apple/swift-syntax.git
+                  module: SwiftSyntax
+                  rule:
+                      upToNextMinor:
+                          from: 0.50600.1
```

Comment:
I'm not sure about the about the other examples but I think they are fine, but can we switch SwiftSyntax out for something a little more lightweight since CI has to clone and compile this package and its dependencies. 

That being said, if it doesn't take too much time then it's fine but last I remember, it was big 😅